.travis.yml

```diff
@@ -5,7 +5,8 @@ services:
 
 language: node_js
 
-node_js: node
+node_js: 
+  - 16
```

Comment:
**Callout**: Currently there is an issue with the latest version (node18)  requiring  dependencies that are not built with travis build machine. Dropping to v16 for compatibility reasons.